.github/workflows/ci.yml

```diff
@@ -60,9 +60,21 @@ jobs:
           - "json:lint"
           - "php:sniff"
           - "php:codestyle"
-          - "php:copypaste"
```

Comment:
Please let's modify the corresponding Composer command instead of adding a new job.